keadm/cmd/keadm/app/cmd/common/types.go

```diff
@@ -28,6 +28,7 @@ type InitOptions struct {
 	KubeConfig       string
 	Master           string
 	AdvertiseAddress string
+	DNS              string
```

Comment:
Does the `keadm` command has the `DNS` flag option?

Reply:
@fisherxu  yes, in another pr.

Reply:
We can add it to this one.

Reply:
@fisherxu  ok, i will merge them later.